src/transpose/index.js

```diff
@@ -4,6 +4,25 @@
  * @param {number[]} array The array to transpose
  * @returns {number[]} The transposed array
  */
-function transpose(array) {}
-
+function transpose(array) {
+     if(array.length < 1 || array.length > 10) {
+      return;
+    } 
+    if(array[0].length < 1 || array[0].length > 10) {
+      return;
+    }
```

Comment:
Why limit the array's length to less than 10?

That's a hardcode that you have to remove. We want our arrays to have a dynamic length.